owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/session.go

```diff
@@ -113,8 +113,11 @@ func RegisterUser(c echo.Context) error {
 	err = db.RegisterUser(userData)
 	if err != nil {
 		// could not register this user into MongoDB (or MongoDB err connection)
-		return c.JSON(http.StatusBadRequest, map[string]string{"result": "error", "details": "Error user data2."})
+		// return c.JSON(http.StatusBadRequest, map[string]string{"result": "error", "details": "Error user data2."})
```

Comment:
I think we don't need this comment anymore, would you mind changing or removing it?

Reply:
Thank you, @Krlier. Did on b1316fa, can you check it, please?